src/OpenTK.Windowing.Common/EventingModels/FocusedChangedEventArgs.cs

```diff
@@ -0,0 +1,36 @@
+//
+// FocusChangedEventArgs.cs
+//
+// Copyright (C) 2018 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+using System;
+
+namespace OpenToolkit.Windowing.EventingModels
+{
+    /// <summary>
+    /// Defines the event data for the window focus changing.
+    /// </summary>
+    public class FocusedChangedEventArgs : EventArgs
+    {
+        public FocusedChangedEventArgs()
+        {
+            
+        }
+
+        public FocusedChangedEventArgs(FocusedChangedEventArgs other)
```

Comment:
Do we even need copy constructors? They don't seem to be in use, and I currently can't think of a reason we would need one?

Reply:
I doubt we need them.

Reply:
thanks, but I just now found that one:
```
    /// Do not cache instances of this type outside their event handler.
    /// If necessary, you can clone an instance using the
    ///  <see cref="MouseButtonEventArgs(MouseButtonEventArgs)" /> constructor.
```
So my guess is that we perhaps want to reuse the EventArgs(MemoryPooling), which would make sense, as they can produce a lot of garbage for the GC. But currently it is not implemented yet though...